main.py

```diff
@@ -0,0 +1,92 @@
+from fastapi import FastAPI, Request
+import uvicorn
+from cryptography.fernet import Fernet
+from os import popen
+
+app = FastAPI()
+from fastapi.responses import StreamingResponse
+
+
+def load_key():
+    """
+    Load the previously generated key
```

Comment:
What is `previously`? You open a file and return the key inside of it. The function doesn't care that it was `previously` generated.